packages/golden_toolkit/lib/src/testing_tools.dart

```diff
@@ -151,12 +151,13 @@ Future<void> screenMatchesGolden(
     fail(
         'Golden tests MUST be run within a testGoldens method, not just a testWidgets method. This is so we can be confident that running "flutter test --name=GOLDEN" will run all golden tests.');
   }
+  final pumpAfterPrime = customPump ?? _onlyPumpAndSettle;
```

Comment:
minor... but now that this doesn't need to be compile time constant, it might be more clear to just in-line the func